src/service/visitsService.js

```
@@ -0,0 +1,52 @@
+import axiosInstance from "./axiosInstance";
+
+class VisitsService {
+  static async saveHomeVisits(formData) {
+    try {
+      const response = await axiosInstance.post(`/visits`, {
+        ...formData,
+      });
+      return response.data;
+    } catch (error) {
+      throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)